src/Simulations/run.jl

```diff
@@ -117,6 +117,13 @@ end
 
 const ModelCallsite = Union{TendencyCallsite, UpdateStateCallsite}
 
+""" Step `sim`ulation forward by Δt. """
+function time_step!(sim::Simulation, Δt)
+    sim.Δt = Δt
+    sim.align_time_step = false # ensure Δt
```

Comment:
I'm wondering whether `sim.align_time_step` should be returned to its original state after `time_step!`?
Or whether an info is printed out to the user?

```suggestion
    if sim.align_time_step == true
        sim.align_time_step = false # ensure Δt
        @info "sim.align_time_step set to false"
    end
```

Reply:
hm... on second thought this might be a bit overhead

Reply:
I think that would make it harder to use not easier

Reply:
As for returning it to the original value, I think that might be harder to understand. I like that this is simple

Reply:
fair

tbh I'm not sure what's the intention; at the moment I only use `time_step!(model)` for debugging or something